sdk/src/oauth/state-machines/debug-oauth-2025-06-18.ts

```
@@ -604,18 +617,11 @@ export const createDebugOAuthStateMachine = (
-              // Parse WWW-Authenticate header to extract resource_metadata URL
-              // Format: Bearer resource_metadata="https://example.com/.well-known/oauth-protected-resource"
-              const resourceMetadataMatch = state.wwwAuthenticateHeader.match(
-                /resource_metadata="([^"]+)"/,
-              );
-              if (resourceMetadataMatch) {
-                extractedResourceMetadataUrl = resourceMetadataMatch[1];
-              }
-            }
+            const challengeParams = parseBearerAuthenticateParameters(
+              state.wwwAuthenticateHeader,
+            );
```

Biome (2.4.10)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)